js/samples/04.ai-apps/h.datasource-azureOpenAI/README.md

```diff
@@ -31,20 +31,15 @@ This guide will show you have the set up your custom copilot for Teams using Azu
-| [Microsoft Teams](https://www.microsoft.com/microsoft-teams/download-app)                                                     | Microsoft Teams to collaborate with everyone you work with through apps for chat, meetings, and call-all in one place.                                                                                                                                              |
-| [Microsoft&nbsp;Edge](https://www.microsoft.com/edge) (recommended) or [Google Chrome](https://www.google.com/chrome/)        | A browser with developer tools.                                                                                                                                                                                                                                     |
-| [Microsoft 365 developer account](/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant)                  | Access to Teams account with the appropriate permissions to install an app and [enable custom Teams apps and turn on custom app uploading](../../../concepts/build-and-test/prepare-your-o365-tenant.md#enable-custom-teams-apps-and-turn-on-custom-app-uploading). |
-| [Yarn](https://yarnpkg.com/) (1.22.x or greater)                                                                              | Node.js package manager used to install dependencies and build samples.                                                                                                                                                                                             |
-| [Azure OpenAI](https://oai.azure.com/portal)                                                                                  | Deploy Azure OpenAI large language models and test your custom copilot ideas in the Azure OpenAI Studio Playground. If you want to host your app or access resources in Microsoft Azure, you must create an Azure OpenAI service.                                                                                                                                                                                                                                    |
+| [Microsoft Teams](https://www.microsoft.com/microsoft-teams/download-app)                                                     | Access to a Microsoft Teams account with the appropriate permissions to install an app, [enable custom Teams apps, and turn on custom app uploading.](https://learn.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant#enable-custom-teams-apps-and-turn-on-custom-app-uploading)                                                                                                                                             |
```

Comment:
They would still need a developer account right?

Reply:
added back in with the latest commit